### DISCUSSION TITLE
Getting the API Key

### DISCUSSION THREAD

**Discussion by ktm-p** [tusharlock10/paladins-edge-client#460] [Q&A] [answered]
Out of curiosity, how did you obtain the API key to use when developing PaladinsEdge? The form seems to have closed.

**Comment by tusharlock10** [comment, marked as answer]
The platform for obraining API keys is shut, and they have already told that no new API keys will be provided. I obtained my keys somewhere in 2021, when there were some Hi-Rez developers dedicated for maintaining the API, all of which have left the company now.

For what purpose do you want a key? Maybe I could share one of my keys with you, if you are planning to create somthing good
